operator/templates/stateful-set.yaml

```diff
@@ -62,6 +62,16 @@ spec:
                 command:
                   - nodetool
                   - drain
+          readinessProbe:
+            exec:
+              command:
+                  - /bin/bash
+                  - /etc/cassandra/cassandra-readiness-probe.sh
+            initialDelaySeconds: {{ .Params.READINESS_CHECK_INITIAL_DELAY }}
+            periodSeconds: {{ .Params.READINESS_CHECK_PERIOD }}
+            timeoutSeconds: {{ .Params.READINESS_CHECK_TIMEOUT }}
+            successThreshold: {{ .Params.READINESS_SUCCESS_THRESHOLD }}
```

Comment:
I wonder if this param is even needed to be configurable since we only need 1 successful check. But in the spirit of configuring :all-the-things:...

Reply:
I've exposed all properties which k8s offer for readiness/liveness probe. Maybe it will be used :)